types/web-element.d.ts

```diff
@@ -361,8 +361,8 @@ export type ScopedElementRect = {
 };
 
 export type DragAndDropDestination = {
-  readonly xOffset: number;
-  readonly yOffset: number;
+  readonly x: number;
+  readonly y: number;
```

Comment:
The destination could also be a `WebElement`. We should consider that as well here.

Reply:
Perhaps you could just help the other person with their PR?

Reply:
👍